assets/src/scss/style.scss

```diff
@@ -38,6 +38,7 @@ Text Domain: planet4-master-theme
 @import "layout/blocks";
 @import "layout/cookies";
 @import "layout/cookies-new";
+@import "layout/country-selector";
```

Comment:
Maybe we should look for a way to split this conditional CSS from the main CSS? If you see above it we also have `cookies-new`, and we may want to add more things behind a feature toggle.

There's a relatively high performance penalty because this is all blocking CSS.

Could be nice to have a mechanism that avoids this problem, but might be complicated to implement, as each separate file needs to have a entrypoint to build it.

Reply:
I created a new style-new.scss file for the navigation bar implementation, because there's too many conflicts with the current CSS that are too hard to avoid, and it's a bigger file. I'll look for a more maintainable way to do that in the navigation bar PR

Reply:
Nice :+1: Maybe we can set up code splitting similar to how our blocks work? We currently only do it for blocks, but there's a lot of areas where we can apply it. And it potentially allows us to avoid the majority of the current blocking CSS in most cases.

Reply:
For example here it would allow us to avoid duplicating the main entrypoint. But also afterwards we could exclude all this CSS if the minimal navigation is used.

Reply:
Just a quick comment on the performance aspect. We don't plan to keep that under a feature flag for very long. Actually it would be activated by default in almost all NROs. It would only be a handful of NROs that would need to update their child theme markup before they toggle the switch.

Reply:
@comzeradd I ended up doing [a PR to set this up](https://github.com/greenpeace/planet4-master-theme/pull/1521), after seeing that having both variants loaded at the same time caused specificity issues, which would force us to make the new CSS more complex.

Also it allowed us to merge this PR earlier without affecting production CSS payload size.